src/main.cpp

```diff
@@ -140,13 +137,17 @@ void loop() {}  // loop() runs on core 1
 }
 
 [[noreturn]] void taskBleCommunication(void *pvParameters) {
-  Serial.println("Task 'Bluetooth transmission' running on core ");
+  Serial.print("Task 'Bluetooth transmission' running on core ");
   Serial.println(xPortGetCoreID());
   GloveMeasurements glove_measurements;
   for (;;) {
     for (int i = 0; i < kQueueSize; i++) {
-      xQueueReceive(queue, &glove_measurements, portMAX_DELAY);
-      bleCommunicator.sendMeasurements(glove_measurements);
+      if (xQueueReceive(queue, &glove_measurements, portMAX_DELAY) == pdPASS) {
+        bleCommunicator.sendMeasurements(glove_measurements);
```

Comment:
Si muevo la linea 142 dentro del loop for, para que siempre sea una nueva instancia me da el siguiente error: ```pGuru Meditation Error: Core  1 panic'ed (LoadProhibited). Exception was unhandled```

Reply:
Ok, interesante, habría que ponerse a estudiar el problema, pero no da para ello ahora. Tal vez una idea mejor sería allocar un gloveMeasurements en el heap acá, total el manejo del heap en esta situación estaría bien contenida adentro de esta función y no sería tanto un problema. Pero bueno ya lo resolviste de otra manera.